package.json

```diff
@@ -41,9 +42,32 @@
   "main": "index.js",
   "scripts": {
     "test:gen:2.0": "rm -rf dist && mkdir dist && cd dist && yo symphony 2.0",
+    "jest": "jest --collectCoverage",
```

Comment:
I would also suggest to use standard scripts like test as to avoid users to guess which ones to use (not sure if there are conventions but test seems pretty common)

Reply:
https://www.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/ seems to suggest scripts like test/clean/lint